config/environments/development.rb

```diff
@@ -12,6 +12,8 @@
   # Show full error reports.
   config.consider_all_requests_local = true
 
+  config.active_storage.service = :local
```

Comment:
I like that you used active_storage so it's stored locally, maybe later we could implement the storage to a server etc.